From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Stefan Reiter <s.reiter@proxmox.com>
Date: Thu, 10 Jun 2021 10:50:22 +0200
Subject: [PATCH] remove reference to libudev

since there's no command line flag I can see...

Signed-off-by: Stefan Reiter <s.reiter@proxmox.com>
[ SI adapt to aebd94cc8541e0ec3b1de57edbd57c4280213089 ]
Signed-off-by: Stoiko Ivanov <s.ivanov@proxmox.com>
---
 config/user-libudev.m4 | 17 -----------------
 config/user.m4         |  1 -
 2 files changed, 18 deletions(-)
 delete mode 100644 config/user-libudev.m4

diff --git a/config/user-libudev.m4 b/config/user-libudev.m4
deleted file mode 100644
index 8c3c1d7e0..000000000
--- a/config/user-libudev.m4
+++ /dev/null
@@ -1,17 +0,0 @@
-dnl #
-dnl # Check for libudev - needed for vdev auto-online and auto-replace
-dnl #
-AC_DEFUN([ZFS_AC_CONFIG_USER_LIBUDEV], [
-	ZFS_AC_FIND_SYSTEM_LIBRARY(LIBUDEV, [libudev], [libudev.h], [], [udev], [], [user_libudev=yes], [user_libudev=no])
-
-	AS_IF([test "x$user_libudev" = xyes], [
-	    AX_SAVE_FLAGS
-
-	    CFLAGS="$CFLAGS $LIBUDEV_CFLAGS"
-	    LIBS="$LIBUDEV_LIBS $LIBS"
-
-	    AC_CHECK_FUNCS([udev_device_get_is_initialized])
-
-	    AX_RESTORE_FLAGS
-	])
-])
diff --git a/config/user.m4 b/config/user.m4
index 6ec27a5b2..46244f19b 100644
--- a/config/user.m4
+++ b/config/user.m4
@@ -14,7 +14,6 @@ AC_DEFUN([ZFS_AC_CONFIG_USER], [
 	AM_COND_IF([BUILD_LINUX], [
 		ZFS_AC_CONFIG_USER_UDEV
 		ZFS_AC_CONFIG_USER_SYSTEMD
-		ZFS_AC_CONFIG_USER_LIBUDEV
 		ZFS_AC_CONFIG_USER_LIBUUID
 		ZFS_AC_CONFIG_USER_LIBBLKID
 	])
